src/styles/features/view/sidebarMenu.css

```diff
@@ -21,212 +21,253 @@
       color: var(--link-visited);
     }
   }
+}
 
-  & .sidebarSelect {
-    position: absolute;
-    /* z-index: 200; */
-    top: calc(50% - 6em);
-    left: -3em;
-    height: 12em;
-    width: 3em;
-    display: flex;
-    flex-direction: column;
-    border-color: black;
-    border-style: solid;
-    border-width: 2px 0px 2px 2px;
-    /* box-sizing: border-box; */
-    border-radius: 3px 0px 0px 3px;
-    box-shadow: -8px 0px 35px 0px rgba(0,0,0,0.4);
+.sidebarSelect {
```

Comment:
CSS always uses dash case, e.g. `foo-bar` --- not `fooBar`.

Reply:
What's the reason for this (out of curiosity)?

Reply:
HTML is largely case insensitive, for one.  Another is that it's just the convention.  Just like using PascalCase for classes in languages like JS or Java.

Reply:
Fix made in this PR and merged